lexer/stateful/stateful.go

```diff
@@ -145,15 +290,38 @@ type Definition struct {
 	backrefCache sync.Map
 }
 
+// MustSimple creates a new lexer definition based on a single state described by `rules`.
```

Comment:
Remove this and `Must()` below, there's an existing `lexer.Must()` which I'd like to keep consistent.

Reply:
Actually, disregard this I think I prefer the opposite direction - add Must* helpers for all concrete lexers and remove the shared one. It's more convenient.

Reply:
Ok